src/List/ListItemAction.js

```diff
@@ -0,0 +1,47 @@
+import React, { PropTypes, Children, cloneElement } from 'react';
+import classNames from 'classnames';
+
+class ListItemAction extends React.Component {
+    static propTypes = {
+        children: PropTypes.element.isRequired,
+        className: PropTypes.string,
+        info: PropTypes.string
+    };
+
+    static defaultProps = {
+        info: null
+    };
+
+    render() {
+        const { className, info } = this.props;
+
+        const classes = classNames('mdl-list__item-secondary-content', className);
+
+        let { children } = this.props;
+
+        try {
+            const child = Children.only(children);
+            children = cloneElement(child, {
```

Comment:
`const child = cloneElement(Children.only(children), {...`
No need to override children.